src/app/demo/typescript/typescript.component.ts

```diff
@@ -1,6 +1,6 @@
-import {Component} from '@angular/core';
-import {typeScriptWithConsoleLog, withDeps} from '../../exercise/helpers/helpers';
-import {ng2tsConfig} from '../../../../ng2ts/ng2ts';
+import { Component, HostBinding } from '@angular/core';
```

Comment:
We mostly don't have spaces around imports (and I totally see this being reformatted back and forth). 
Is there an option in VS code which allows to configure that?